.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     environment: BUF
     needs: run_checker
-    if: ${{ needs.run_checker.outputs.run_release_proto }} && github.repository_owner == 'astriaorg'
+    if: needs.run_checker.outputs.run_release_proto && github.repository_owner == 'astriaorg'
```

Comment:
The first conditional need not be `== 'true'` like in the other `if`s of this patch?

EDIT: I think this might. In a similar && condition you have == further down